lib/cmds/login.ts

```diff
@@ -33,9 +34,20 @@ module.exports.builder = yargs => {
         'Copyright 2019 Contentful'
       ].join('\n')
     )
+
+interface Context {
+  managementToken?: string
+}
+
+interface LoginProps {
+  context: Context
+  managementToken: string
 }
 
-async function login({ context, managementToken: managementTokenFlag }) {
+export const login = async ({
+  context,
+  managementToken: managementTokenFlag
```

Comment:
what the difference between `managementToken` from context and `managementTokenFlag` from props?

Reply:
Context => to check and get if there's a token in `.contentfulrc.json`
Flag => gets the token from command options flag when users define it

<img width="588" alt="Screenshot 2022-08-11 at 15 49 01" src="https://user-images.githubusercontent.com/6163988/184148633-91c61b30-ed6f-4f97-ae9a-4b82e11e7206.png">